website/docs/d/bfd_multihop_node_policy.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: -
+layout: "aci"
+page_title: "ACI: aci_bfdmultihop_node_policy"
+sidebar_current: "docs-aci-data-source-bfdmultihop_node_policy"
+description: |-
+  Data source for ACI BFD Multihop Node Policy
+---
+
+# aci_bfdmultihop_node_policy #
+
+Data source for ACI BFD Multihop Node Policy
+
+
+## API Information ##
+
+* `Class` - bfdMhNodePol
+* `Distinguished Name` - uni/tn-{name}/bfdMhNodePol-{name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Policies -> Protocol -> BFD Multihop -> Node Policies
+
+
+## Example Usage ##
+
+```hcl
+data "aci_bfdmultihop_node_policy" "example" {
+  tenant_dn  = aci_tenant.example.id
+  name  = "example"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the BFD Multihop Node Policy object.
```

Comment:
Should these include Type?

Reply:
yes, it should. I think it would be better for consistency.